lib/cc/service/http.rb

```diff
@@ -21,10 +21,10 @@ def service_get(url = nil, body = nil, headers = nil, &block)
     raw_get(url, body, headers, &block)
   end
 
-  def service_post(url, body = nil, headers = nil, &block)
+  def service_post(url, body = nil, headers = nil, response_prefix = nil, &block)
```

Comment:
I think it's a big-ish refactor to remove the response-formatting from the http code. I'm open to doing it, though, as it will make it easier to make changes like this one which accommodate multiple request metadata in one service response